CodeReview/src/commands/CCodeReview.cpp

```diff
@@ -51,6 +51,9 @@ using namespace Visualization;
 
 namespace CodeReview {
 
+const QString CCodeReview::SAVE_COMMAND = "save";
```

Comment:
we already use the `save` command for ordinary save. Let's pick a different name. `save-review`